packages/core/package.json

```diff
@@ -49,7 +49,7 @@
     "url": "https://github.com/KnapsackPro/knapsack-pro-js/issues"
   },
   "dependencies": {
-    "axios": "^0.27.2",
```

Comment:
⚠️ We need to be careful here. Updating axios in the past caused breaking retry reqeusts mechanism that's handled by axios-retry

https://github.com/KnapsackPro/knapsack-pro-core-js/pull/15
https://github.com/KnapsackPro/knapsack-pro-core-js/pulls?q=is%3Apr+is%3Aclosed+axios-retry

We would have to check first if the API is not reachable (use a fake URL in a [bin script](https://github.com/KnapsackPro/knapsack-pro-js/blob/main/packages/jest-example-test-suite/bin/knapsack_pro_jest)) and check that after 3 requests the Fallback Mode is started.